### PR TITLE
Restructure cvs

### PR DESCRIPTION
**Pull request by InnocentBug**

### CODE REVIEW

File: pysages/ssages/collective_variables/shape.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    def function(self):
+        return acylindricity
+
+
+def acylindricity(positions):
+    λ1, λ2, _ = principal_moments(positions)
+    return (λ2 - λ1)
+
+
+class ShapeAnysotropy(CollectiveVariable):
```
Comment (InnocentBug) [inline pysages/ssages/collective_variables/shape.py:84]: Is this a typo and supposed to be `ShapeAnisotropy`?
same for the function below

Comment (InnocentBug) [reply pysages/ssages/collective_variables/shape.py]: ok, fixed the typo

File: pysages/ssages/collective_variables/__init__.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020: Pablo Zubieta (see LICENSE.md)
+# Copyright (c) 2021: Ludwig Schneider (see LICENSE.md)
+
+# pylint: disable=unused-import,relative-beyond-top-level
+# flake8: noqa F401
+
+from .angles import Angle, DihedralAngle
+from .shape import RadiusOfGyration, PrincipalMoment, Asphericity, Acylindricity, ShapeAnisotropy
```
Comment (pabloferz) [inline pysages/ssages/collective_variables/__init__.py:9]: For API exposure I like to have long list of imports on a tuple spanning multiple lines, so it's easier to track in the commits what is added/removed

Comment (InnocentBug) [reply pysages/ssages/collective_variables/__init__.py]: agreed, and implemented

File: pysages/ssages/collective_variables/__init__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    DihedralAngle,
+)
+
+from .shape import (
+    RadiusOfGyration,
+    PrincipalMoment,
+    Asphericity,
+    Acylindricity,
+    ShapeAnisotropy,
+    )
```
Comment (pabloferz) [inline pysages/ssages/collective_variables/__init__.py:19]:
```suggestion
)
```

File: pysages/ssages/collective_variables/__init__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+from .coordinates import (
+    Component,
+    Distance,
+    )
```
Comment (pabloferz) [inline pysages/ssages/collective_variables/__init__.py:24]:
```suggestion
)
```

**Review by pabloferz — APPROVED** [review]

**Pull request merged**